[12/09/19 - 17:19:16] Incapable de créer le FIFO (côté serveur) [ Under : /cygdrive/c/Users/fsiga/Desktop/OS/server/server.c in function main (line 23) ]
[12/19/19 - 17:02:33] Incapable d'établir la connexion au descripteur de fichier (côté client) [ Under : /cygdrive/c/Users/fsiga/Desktop/OS_I2/client/client.c in function main (line 57) ]
[12/19/19 - 17:04:46] Cannot access the public file descriptor, the server could have been interrupted ... [ Under : /cygdrive/c/Users/fsiga/Desktop/OS_I2/client/client.c in function sendMessageToServerBeforeLeaving (line 92) ]
[12/19/19 - 17:06:59] Cannot connect to the public file descriptor (server side) [ Under : /cygdrive/c/Users/fsiga/Desktop/OS_I2/server/server.c in function main (line 47) ]
[12/19/19 - 17:11:02] Incapable d'établir la connexion au descripteur de fichier (côté client) [ Under : /cygdrive/c/Users/fsiga/Desktop/OS_I2/client/client.c in function main (line 57) ]
[12/19/19 - 17:11:16] Incapable d'établir la connexion au descripteur de fichier (côté client) [ Under : /cygdrive/c/Users/fsiga/Desktop/OS_I2/client/client.c in function main (line 57) ]
[12/19/19 - 17:26:15] Cannot access the public file descriptor, the server could have been interrupted ... Bye ! [ Under : /cygdrive/c/Users/fsiga/Desktop/OS_I2/client/client.c in function main (line 58) ]
[12/19/19 - 17:28:57] Clients list is empty (server side) [ Under : /cygdrive/c/Users/fsiga/Desktop/OS_I2/server/structClient.c in function supprimer (line 68) ]
[12/19/19 - 17:30:40] Cannot access the public file descriptor, the server could have been interrupted ... Bye ! [ Under : /cygdrive/c/Users/fsiga/Desktop/OS_I2/client/client.c in function main (line 58) ]
[12/19/19 - 17:30:43] Cannot access the public file descriptor, the server could have been interrupted ... Bye ! [ Under : /cygdrive/c/Users/fsiga/Desktop/OS_I2/client/client.c in function sendMessageToServerBeforeLeaving (line 94) ]
[12/19/19 - 17:35:13] The server is not running (client side) [ Under : /cygdrive/c/Users/fsiga/Desktop/OS_I2/client/client.c in function sendMyPidToTheServer (line 119) ]
[12/19/19 - 19:43:14] Cannot access the public file descriptor, the server could have been interrupted ... Bye ! [ Under : /cygdrive/c/Users/fsiga/Desktop/OS_I2/client/client.c in function sendMessageToServerBeforeLeaving (line 96) ]
[12/19/19 - 19:49:07] Cannot access to the private file descriptor (client side) [ Under : /cygdrive/c/Users/fsiga/Desktop/OS_I2/client/client.c in function main (line 71) ]
[12/19/19 - 19:49:07] Cannot connect to the private file descriptor (server side) [ Under : /cygdrive/c/Users/fsiga/Desktop/OS_I2/server/server.c in function main (line 94) ]
[11/12/19 - 18:37:31] Incapable de créer le PRIVATE FIFO (côté client) [ Under : /home/pi/dev/projet/client/client.c in function main (line 42) ]
[11/12/19 - 18:39:45] The server is not running (client side) [ Under : /home/pi/dev/projet/client/client.c in function sendMyPidToTheServer (line 121) ]
[11/12/19 - 18:42:34] Clients list is empty (server side) [ Under : /home/pi/dev/projet/server/structClient.c in function supprimer (line 68) ]
[11/12/19 - 18:46:26] Cannot access the public file descriptor, the server could have been interrupted ... Bye ! [ Under : /home/pi/dev/projet/client/client.c in function sendMessageToServerBeforeLeaving (line 96) ]
[11/12/19 - 18:48:01] Clients list is empty (server side) [ Under : /home/pi/dev/projet/server/structClient.c in function supprimer (line 68) ]
[11/12/19 - 18:52:10] Clients list is empty (server side) [ Under : /home/pi/dev/projet/server/structClient.c in function supprimer (line 68) ]
[11/12/19 - 18:54:44] Clients list is empty (server side) [ Under : /home/pi/dev/projet/server/structClient.c in function supprimer (line 68) ]
[12/20/19 - 16:21:25] Clients list is empty (server side) [ Under : /home/pi/dev/projet/server/structClient.c in function supprimer (line 68) ]
[12/20/19 - 16:23:07] Clients list is empty (server side) [ Under : /home/pi/dev/projet/server/structClient.c in function supprimer (line 68) ]
[12/20/19 - 18:52:52] Cannot access the public file descriptor, the server could have been interrupted ... Bye ! [ Under : /home/pi/dev/projet/client/client.c in function main (line 59) ]
[12/20/19 - 18:54:10] Cannot access the public file descriptor, the server could have been interrupted ... Bye ! [ Under : /home/pi/dev/projet/client/client.c in function main (line 51) ]
[12/20/19 - 20:35:23] Error while trying to open the questions/reponses file (server side) [ Under : /home/pi/dev/projet/server/dialogue.c in function line_number (line 54) ]
[12/20/19 - 20:35:49] Error while trying to open the questions/reponses file (server side) [ Under : /home/pi/dev/projet/server/dialogue.c in function line_number (line 54) ]
[12/20/19 - 20:35:56] Error while trying to open the questions/reponses file (server side) [ Under : /home/pi/dev/projet/server/dialogue.c in function line_number (line 54) ]
[12/20/19 - 21:50:39] Cannot access the public file descriptor, the server could have been interrupted ... Bye ! [ Under : /home/pi/dev/projet/client/client.c in function main (line 58) ]
[12/20/19 - 22:01:48] Cannot access the public file descriptor, the server could have been interrupted ... Bye ! [ Under : /home/pi/dev/projet/client/client.c in function main (line 58) ]
[12/20/19 - 22:05:19] The server is not currently running [ Under : /home/pi/dev/projet/client/client.c in function main (line 91) ]
[12/20/19 - 22:05:31] The server is not currently running [ Under : /home/pi/dev/projet/client/client.c in function main (line 91) ]
[12/20/19 - 22:05:31] The server is not currently running [ Under : /home/pi/dev/projet/client/client.c in function main (line 91) ]
[12/20/19 - 22:33:28] The server is not currently running [ Under : /home/pi/dev/projet/client/client.c in function main (line 94) ]
[12/20/19 - 23:31:26] The server is not currently running [ Under : /home/pi/dev/projet/client/client.c in function main (line 98) ]
[12/20/19 - 23:43:54] The server is not currently running [ Under : /home/pi/dev/projet/client/client.c in function main (line 95) ]
[12/20/19 - 23:24:50] The server is not currently running [ Under : /home/pi/dev/projet/client/client.c in function main (line 95) ]
[12/20/19 - 23:32:14] The server is not currently running [ Under : /home/pi/dev/projet/client/client.c in function main (line 95) ]
[12/20/19 - 23:32:29] The server is not currently running [ Under : /home/pi/dev/projet/client/client.c in function main (line 95) ]
[12/20/19 - 23:34:34] The server is not currently running [ Under : /home/pi/dev/projet/client/client.c in function main (line 95) ]
[12/20/19 - 23:35:03] Cannot access the public file descriptor, the server could have been interrupted ... Bye ! [ Under : /home/pi/dev/projet/client/client.c in function main (line 62) ]
[12/20/19 - 23:36:31] Cannot access the public file descriptor, the server could have been interrupted ... Bye ! [ Under : /home/pi/dev/projet/client/client.c in function main (line 62) ]
[12/20/19 - 23:36:43] Cannot access the public file descriptor, the server could have been interrupted ... Bye ! [ Under : /home/pi/dev/projet/client/client.c in function main (line 62) ]
[12/20/19 - 23:50:05] The server is not currently running .. [ Under : /home/pi/dev/projet/client/client.c in function main (line 97) ]
[12/21/19 - 03:07:48] The server is not currently running .. [ Under : /home/pi/dev/projet/client/client.c in function main (line 97) ]
[12/21/19 - 04:08:01] A server is already open [ Under : /home/pi/dev/projet/server/server.c in function main (line 120) ]
[12/21/19 - 04:08:11] A server is already open [ Under : /home/pi/dev/projet/server/server.c in function main (line 120) ]
